README.md

```diff
@@ -20,11 +20,23 @@ The library includes:
 
 ## Installation
 
-To use the library, add the following lines to your `lakefile.lean`:
+-  Add the following lines to your `lakefile.lean`:
 
-` require rust-lean-models from git "https://github.com/model-checking/rust-lean-models" `
+    ` require «rust-lean-models» from git "https://github.com/model-checking/rust-lean-models" `
 
-Then do `lake update` in the terminal.
+- Change the lean version in `lean-toolchain.lean`:
```

Comment:
We'd like to keep the Lean toolchain version up-to-date, so to avoid updating the README.md every time we update the toolchain version, let's just say "Change the Lean version in `lean-toolchain` to match the version used in this package" or something like that.